package.json

```diff
@@ -874,6 +874,7 @@
         "@microsoft/api-extractor": "^7.33.8",
         "@microsoft/eslint-config-azuretools": "0.2.1",
         "@microsoft/vscode-azext-dev": "^2.0.5",
+        "@types/fs-extra": "^11.0.4",
```

Comment:
Although this worked because `fs-extra` is a transitive dependency, it's a good practice to make it a direct dependency when using it. You can do so with `npm install fs-extra` at the root. This will automatically update `package-lock.json` as well.